enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -86,4 +88,63 @@ public void deleteUser( @Name( "username" ) String username ) throws IllegalCred
         }
         shiroSubject.getUserManager().deleteUser( username );
     }
+
+    @PerformsDBMS
+    @Procedure( "dbms.listUsers" )
+    public Stream<StringResult> listUsers() throws IllegalCredentialsException, IOException
+    {
+        ShiroAuthSubject shiroSubject = ShiroAuthSubject.castOrFail( authSubject );
```

Comment:
I agree, and we have this on a refactoring card - check the box it if you fix it now: https://trello.com/c/RDSWbUBT/125-easy-clean-up-ideas